helper-hardhat-config.ts

```diff
@@ -1,21 +1,26 @@
 export interface networkConfigItem {
+  name: string
```

Comment:
I don't think this is needed. I'd like to keep the code as similar to the video for now.